package.json

```
@@ -4,7 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "cross-env NODE_ENV=production node dist/index.js",
+    "dev": "cross-env NODE_ENV=development ts-node-dev src/index.ts",
+    "build": "tsc -p .",
+    "postbuild": "npm run build && cd client/ && npm i && npm run build",
```

this script makes a problem when we run the `build` script, so maybe we could change it's name to `deployment-postbuild`
```suggestion
    "deployment-postbuild": "npm run build && cd client/ && npm i && npm run build",
```